rest-service/manager_rest/test/test_app.py

```diff
@@ -0,0 +1,33 @@
+#########
+# Copyright (c) 2016 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+
+"""Test the basic HTTP interface, app-wide error handling
+"""
+
+from nose.plugins.attrib import attr
+
+from manager_rest.test import base_test
+
+
+@attr(client_min_version=1, client_max_version=base_test.LATEST_API_VERSION)
+class AppTestCase(base_test.BaseServerTestCase):
```

Comment:
This didn't really fit anywhere else :( I guess just making a new file is fine, and in the future we can test other app-wide things like errorhandlers etc here.